plugins/woocommerce-blocks/packages/components/radio-control/option.tsx

```diff
@@ -47,7 +47,6 @@ const Option = ( {
 				onChange={ onChangeValue }
 				checked={ checked }
 				aria-describedby={ clsx( {
-					[ `${ name }-${ value }__label` ]: label,
 					[ `${ name }-${ value }__secondary-label` ]: secondaryLabel,
 					[ `${ name }-${ value }__description` ]: description,
 					[ `${ name }-${ value }__secondary-description` ]:
```

Comment:
The `secondaryLabel`, `description`, and `secondaryDescription` references have not been removed since they point to valid DOM elements:
![image](https://github.com/user-attachments/assets/373aa1d8-ff81-4903-aa6e-77bb357511b0)